### PR TITLE
Feat: Add `cssAttributes` `jsAttributes` options (resolve #5)

### LINKED ISSUE #5: Add attributes to generateJSReferences
I was trying to add some attributes in my script tag and I had to customize `generateJSReferences` function:

```js
function generateJSReferences(files = [], publicPath = '', attributes = '') {
  return files
    .map(file => `<script src="${publicPath}${file}" ${attributes}></script>`)
    .join('');
}
```

I can call it in my custom template like so: `${generateJSReferences(js, publicPath, 'defer="defer"')}`.

Would be nice to have this functionality by default in *mini-html-webpack-plugin*. What do you think? You can do the same for `generateCSSReferences`.

### PR DESCRIPTION
**Pull request by pldg**
- Add `generateAttributes` utility method
- Add test for `jsAttributes`
- Refactoring `generateCSSReferences` `generateJSReferences`
- Update readme to reflect the changes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -109,23 +106,26 @@ function defaultTemplate({
...
 		.join('');
 }
 
 function generateJSReferences({
 	files = [],
 	publicPath = '',
-	jsAttributes = '',
+	attributes = {},
 }) {
+	attributes = generateAttributes(attributes);
```
Comment (sapegin) [inline index.js]: Let's no shadow the function argument to make code easier to understand.

Comment (pldg) [reply index.js]: Thanks @sapegin! In the current release *v1.0.0*, `generateCSSReferences` accept two arguments `(css, publicPath)`, in this pull request I changed it to accept an object `({files: css, attributes: cssAttributes, publicPath})`. Same situation for `generateJSReferences`. This is a breaking change for me. Do you think it should be released under *v1.1.0* or *2.0.0*?

**Comment by sapegin** [comment]
Yeah, let's do 2.0.0 to be safe 🔒

**Comment by pldg** [comment]
I propose another change. If you set `output.publicPath` in webpack config this should be reflected correctly in *mini-html-webpack-plugin* instead of manually pass it as option. How do I access webpack `output.publicPath` within the plugin?

**Pull request merged**